tjba/parser.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"encoding/json"
+	"strconv"
+
+	"github.com/dadosjusbr/coletores"
+)
+
+// newtjbaEmployee creates a tjbaEmployee from a map[string]interface{}
+func newTjbaEmployees(payload string) ([]tjbaEmployee, error) {
+	var employees []tjbaEmployee
+	err := json.Unmarshal([]byte(payload), &employees)
+	return employees, err
+}
+
+// employeeType returns employee.Type
+func employeeType(value string) string {
+	// possible values from the source: S (servidor), D (desembargador) and J (juiz)
+	if value == "S" {
+		return "servidor"
+	} else if value == "D" || value == "J" {
+		return "membro"
+	}
+	return ""
+}
+
+// active returns Employee.Active
+func active(value string) bool {
+	// possible values from the source: A (ativo) or I (inativo)
+	return value == "A"
+}
+
+// FromTjbaEmployeeToEmployee convert from TJ-BA employee to coletores.Employee
+func FromTjbaEmployeeToEmployee(tjbaEmployee []tjbaEmployee) []coletores.Employee {
+	var employees []coletores.Employee
+	for i := range tjbaEmployee {
+		var perks = coletores.Perks{Total: tjbaEmployee[i].PerksTotal}
```

Comment:
Sugestão:

```suggestion
		perks := coletores.Perks{Total: tjbaEmployee[i].PerksTotal}
```

Caso ache bacana escrever um pouco menos, pode aplicar nos exemplos abaixo.